modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -564,14 +567,22 @@ export default class AvgRTPStatsReporter {
      */
     _calculateAvgStats(data) {
 
+        if (!data) {
+            logger.error('No stats');
+
+            return;
+        }
+
+        if (!data.transport || !data.transport.length) {
+            logger.error('No transport.');
```

Comment:
What I mean is that some stats are available on one browser while missing on another